### DISCUSSION TITLE
Change accent color

### DISCUSSION THREAD

**Discussion by gravelfreeman** [mealie-recipes/mealie#2543] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

The more time I spend in Mealie the more annoyed I get at the orange color. Since it's self hosted I think it would be great to have an official option to change the accent color to something else.

### Please provide a concise description of the feature that would resolve your issue.

Setting in the options page to change the accent color with a color picker.

### Please consider and list out some caveats or tradeoffs made in your design decision

I think this wouldn't be complicated to integrate instead of a whole theme system.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by michael-genson** [comment]
This should be possible already:

https://nightly.mealie.io/documentation/getting-started/installation/frontend-config/

**Comment by eskahone** [comment]
Dear Michael!
Using the env variables is working and gives Mealie a very nice option to customize the color scheme.
However, the setting of the primary color, does not affect the header of of the (mobile) browser bar or of the PWA.
![IMG_20230926_141634](https://github.com/mealie-recipes/mealie/assets/74679226/455cf5c8-790a-4cff-ab0e-4b446d94b518)

It seems that this color is hard-coded somewhere. I did find the color code of the orange in the PWA section of nuxt.config.js in the frontend folder. Not sure if this can be replaced.

**Reply by gravelfreeman** [reply]
Yeah this one is especially annoying. Still I want to thank @michael-genson for the tip which worked great so far. There's a few things I can't achieve and color is very limited. If anyone's interested in my version of the Dracula theme for Mealie here's what I've used.

![image](https://github.com/mealie-recipes/mealie/assets/44218371/02581bc9-0cb7-4043-b586-4bd42337d524)

I'll update once I have tested more and will add it to my docs.
